### PR TITLE
Mutating attribute

### PR DESCRIPTION
**Pull request by chriseidhof**
I added support for mutating attributes.

Thanks @ilyapuchka for giving me some guidance!

### CODE REVIEW

File: Sourcery/Parsing/FileParser.swift [hunk 1]
```diff
@@ -702,7 +702,7 @@ extension FileParser {
                     var attributeRange = prefix.range(of: $0.description, options: .backwards)
                     // we expect all attributes to be prefixed with `@`
```
Comment (Antondomashnev) [inline Sourcery/Parsing/FileParser.swift]: I think it makes sense to update comment as well 😄

Comment (ilyapuchka) [inline Sourcery/Parsing/FileParser.swift]: Maybe it's time to add property like `requiresAtPrefix` (can't find a better name ^^) in the `Attribute` type and hide this check there? We can also reuse it in `name` property of `Attribute` type.

Comment (Antondomashnev) [inline Sourcery/Parsing/FileParser.swift]: It will clean the code - makes sense 👍

**Comment by chriseidhof** [comment]
Maybe `hasAtPrefix` would be a better name?

**Comment by Antondomashnev** [comment]
> Maybe hasAtPrefix would be a better name?

For me, it sounds good.

**Comment by ilyapuchka** [comment]
I'll merge this, will do this improvement with a next pr as I noticed that we also do not parse `final` attribute

**Comment by ilyapuchka** [comment]
Thanks a lot for your contribution @chriseidhof !

**Pull request merged**